engine/src/main/java/org/terasology/engine/audio/events/PlaySoundEvent.java

```diff
@@ -8,20 +8,29 @@
 import org.terasology.engine.network.BroadcastEvent;
 
 /**
- * This event is sent whenever a sound is to be played.
- * The event is not sent to or played for the client of the entity that sent the event as they have simulated it already.
+ * A <i>trigger event</i> to play a sound.
+ * <p>
+ * The event is not sent to or played for the client of the entity that sent the event as they have simulated it
+ * already.
+ * <p>
+ * The target entities {@link org.terasology.engine.logic.location.LocationComponent LocationComponent} defines at which
+ * location (world coordinates) the sound is played at. If the entity does not have a location, the sound is played as
+ * global effect.
```

Comment:
```suggestion
 * The sound played from the target entity is played from the position defined by the {@link org.terasology.engine.logic.location.LocationComponent LocationComponent}.
 * If the entity does not specify a location, then the sound is played as a global sound effect.
```

maybe something like this?